lib/beaker/options/presets.rb

```diff
@@ -185,6 +185,7 @@ def presets
           :puppetdb_port_nonssl   => 8080,
           :puppetserver_port      => 8140,
           :nodeclassifier_port    => 4433,
+          :cache_files_locally    => true, # TODO change to false in next major version
```

Comment:
@rick would you mind doing that as a separate PR?

I have some questions about that code, so I'd rather not include it in my PR.